Makefile

```diff
@@ -49,6 +49,8 @@ BUILD_OPTS_DEPLOY?=-mod=vendor "-ldflags=$(BUILDINFO) -w -s"
 
 check: lint test ## Run linters and tests
 
+check-windows: lint test-windows ## Run linters and tests on windows
```

Comment:
This is not working for me
```
PS C:\Users\Erson\dmsg> make check-windows 
powershell -Command setx GO111MODULE on; golangci-lint run -c .golangci.yml ./...

SUCCESS: Specified value was saved.
panic: load embedded ruleguard rules: typechecker error: rules/rules.go:4:2: could not import github.com/quasilyte/go-ruleguard/dsl (can't find import: "github.com/quasilyte/go-ruleguard/dsl")

goroutine 1 [running]:
github.com/go-critic/go-critic/checkers.init.10()
        C:/Users/Erson/go/pkg/mod/github.com/go-critic/go-critic@v0.5.7/checkers/checkers.go:55 +0x4a8
make: *** [Makefile:55: lint] Error 1
```

Reply:
Not sure about that, i checked in two separate clean VM, cannot reproduce it.

Reply:
Seems to be related to this issue: https://github.com/golangci/golangci-lint/issues/2190

Reply:
I'm not getting the same error anymore. Not sure how. But we need to remove the comment inside `lint` 
```
PS C:\Users\Erson\dmsg> make check-windows
powershell -Command setx GO111MODULE on; golangci-lint run -c .golangci.yml ./...

SUCCESS: Specified value was saved.
# The govet version in golangci-lint is out of date and has spurious warnings, run it separately
process_begin: CreateProcess(NULL, # The govet version in golangci-lint is out of date and has spurious warnings, run it separately, ...) failed.
make (e=2): The system cannot find the file specified.
make: *** [Makefile:58: lint] Error 2
```

Reply:
I still can't reproduce it. Have you got hmm diffutils installed on windows?

Reply:
Is that the `gcc` compiler?

Reply:
idk which package had it, have had no trouble getting gcc from https://jmeubank.github.io/tdm-gcc/

Reply:
Yes, that's not a `gcc` error. The comment inside the recipe was causing that issue. Everything is working now.